### DISCUSSION TITLE
Select multiple handles and connect to 1 target at once

### DISCUSSION THREAD

**Discussion by charles07castillo** [xyflow/xyflow#3045] [Q&A]
![Screenshot 2023-05-10 at 3 29 58 PM](https://github.com/wbkd/react-flow/assets/120253258/c0c9b926-a57b-4d1f-9746-36eb997bd0f1)

So my 1 Node constructed on multiple handles, I used <Handle component to create it.

How can I allow user to highlight+select multiple handles and then connect it at once

The scenario is, If the user has a 50 handles in 1 node and wants to connect #30 - #40 handles on a single connect

**Comment by moklick** [comment]
This is interesting. I think you are the first person asking this 🤔 Do you want to draw multiple connection lines while connecting as well, or just select handles in one node and then click on another node and add the new connections?

**Comment by charles07castillo** [comment]
What I really want to achieve is you can start draw + selection ( SHIFT + HOLD Left Click) then select the 3 handles. after selecting the 3 handles. I can connect either one of the selected handles, once it connects. all the selected should connect to the same target

**Reply by MFukazawa** [reply]
@charles07castillo 
Did you ever figure out how to do this?
